internal/librarian/generate.go

```diff
@@ -178,6 +182,13 @@ func (r *generateRunner) run(ctx context.Context) error {
 	if err := r.runBuildCommand(ctx, libraryID); err != nil {
 		return err
 	}
+
+	// Push the config changes to GitHub.
+	if r.cfg.PushConfig != "" {
+		if err := commitAndPush(ctx, r.repo, r.ghClient, r.cfg.PushConfig); err != nil {
+			return err
+		}
+	}
```

Comment:
nit: Can you refactor this check inside of `commitAndPush` for consistency with r.runBuildCommand above? The two snippets handle conditional logic differently, makes it harder to read. Also please update the comment accordingly.